### PR TITLE
Feature / Local .env file and Azure deployment support

### PR DESCRIPTION
**Pull request by camalot**
- added support for a `.env` file that can contain key/value pairs for values that are loaded in to environment variables
- added azuredeploy.json template and iisnode.yaml file for deployment to azure
- fixed the background image that had a 1px line that just didn't blend correctly.
- updated `readme.md` to have info about the `.env` and azure deployment
- removed the `0.0.0.0` from the `listen`, as it was causing issues with azure. It will bind to all IP addresses by default.
- added function to handle the normalization of the `PORT`, when it parses it from the `process.env.PORT`

### CODE REVIEW

**Comment by outsideris** [comment]
Thank you for your contribution. It is great to support Azure.
I think variables in `config.js` are used in code. So, give me a time to test in other environments including Azure.

**Comment by camalot** [comment]
I didn't change anything in config.js. just added ability to load environment variables into the config from a .env file (on local) so it works like it would in a server environment.

I see there are now conflicts, I'll check that later and resolve that.

**Comment by outsideris** [comment]
@camalot Yes, I guess the conflict is by https://github.com/outsideris/slack-invite-automation/pull/59 . That is not a big deal. I can merge it manually if you are busy.

File: README.md [hunk 1]
```diff
@@ -9,35 +9,54 @@ and
...
+## Settings
 
-* `community`: your community or team name to display on join page.
-* `slackUrl` : your slack team url (ex: socketio.slack.com)
-* `slacktoken` : access token of slack.
-  You can generate it in <https://api.slack.com/web#auth>.
-  **You should generate the token in admin user, not owner.**
-  If you generate the token in owner user, `missing_scope` error will be occurred.
-* `inviteToken`: an optional security measure - if it is set, then that token will be required to get invited.
-* `locale`: application language (currently `en`, `de`, `es`, `fr`, `pt`, `zh-CN`, `zh-TW`, `ja` and `ko` available).
```
Comment (outsideris) [inline README.md:23]: The reason I mentioned `config.js` is that you removed these lines.

**Comment by camalot** [comment]
Ahhhh, yes, I did, but they were basically redefined. Describing them as the environment variables that they are. I was just trying to define those things without duplicating the definition.

**Comment by outsideris** [comment]
Ok, now this project has many environments to support. So, I have to it more clearly including this PR. Could you remove unnecessary commit(pull request merged commits make it verbose) and a background image?

**Comment by outsideris** [comment]
landed in 6c9f89da186817ae788891c79e55bd3795c310a3

**Pull request closed without merging**